### DISCUSSION TITLE
Participating in the libGDX Community Showcases?

### DISCUSSION THREAD

**Discussion by crykn** [libktx/ktx#367] [General]
Hey @czyzby! 

I don't know if you have noticed, but since the beginning of this year, we have been doing Community Showcases on the official libGDX blog. They are meant to give the creators of interesting community projects the opportunity to present their exciting libraries or tools. I think libKTX would be a great fit for that.

Would you generally be interested in participating in our showcases? Our next free slot for a showcase post would be at the beginning of June. If you have any questions, please feel free to ask either here or on the [libGDX Discord](https://libgdx.com/community/discord/). Our wiki also has some [general information](https://github.com/libgdx/libgdx.github.io/wiki/Community-Showcases).

**Comment by czyzby** [comment]
Hey @crykn, thanks for the suggestion. I've thought about it when I've first seen the project, but I wasn't quite sure if a Kotlin-specific library would be a good fit. The article would probably need an introductory section to Kotlin itself.

I might try to come up with something over the next 2 weekends.

**Reply by crykn** [reply]
Hey @czyzby, how is it going? The June date is over, but July/August would be possible dates if you are interested.

**Reply by czyzby** [reply]
@crykn Busy moving to a new flat, and might participate in a hackathon next week. August is probably more likely, sorry.

**Reply by crykn** [reply]
No problem! We can also move it to another month after August if that suits you better.

**Reply by czyzby** [reply]
I'll just drop an article when I have some free time and feel like writing documentation, so you can post it whenever there's a spot to fill. ;) If there are other projects waiting to be featured, feel free to schedule them for August. I don't want to block a spot for anyone.

**Reply by crykn** [reply]
Sounds like a plan 👍
